src/core/net/ip6_mpl.hpp

```diff
@@ -518,7 +518,7 @@ class Mpl
         kNumSeedEntries = OPENTHREAD_CONFIG_MPL_SEED_SET_ENTRIES,
         kSeedEntryLifetime = OPENTHREAD_CONFIG_MPL_SEED_SET_ENTRY_LIFETIME,
         kSeedEntryLifetimeDt = 1000,
-        kDataMessageInterval = 64
+        kDataMessageInterval = 200
```

Comment:
Thanks @LuDuda for pointing this out. I didn't notice the definition in the spec when analysing. Will recover back to ensure compliant with the spec.

However as far as the case REED_5_2_5 in considered, it involves 18 devices with good connectivity, if following the parameters in the spec, many CCA/NoACK error would be observed on the device side and the retransmission would bring more traffic and increase such error. @jwhui, are there any considerations when defining the parameters DATA_MESSAGE_IMIN/DATA_MESSAGE_IMAX and DATA_MESSAGE_TIMER_EXPIRATIONS in the spec?